monorepo-migration/generate-replace-message-expressions.py

```diff
@@ -57,39 +57,12 @@ def main():
                 )
                 regexes.append(regex)
 
-    # fix up wrong replacements from previous migrations
+    # TBD: Need to create regexes
```

Comment:
Is this a TBD? I had understood from a previous comment we might not need those at all?

Reply:
Actually I have added a regex that is working now, so I can remove it or change it as "add more regexes if needed"...